src/components/picker/helpers/usePickerType.ts

```diff
@@ -42,10 +48,12 @@ const usePickerType = (props: PickerProps) => {
       case PickerModeTypes.Dialog:
       case PickerModeTypes.WheelPicker:
         componentProps.headerProps = props.headerProps;
-        componentProps.dialogProps = componentProps.headerProps && {
-          ...props?.customPickerProps?.dialogProps,
-          headerProps: props.headerProps
-        };
+        componentProps.dialogProps =
+          (componentProps.headerProps && {
```

Comment:
This is very unclear... Why pass customPickerProps? Why only if there are headerProps?